src/wazuh_modules/vulnerability_scanner/src/scanOrchestrator/versionMatcher/versionObjectRpm.hpp

```diff
@@ -161,20 +184,59 @@ class VersionObjectRpm final : public IVersionObject
         /* compared identically but the segment sepparating characters were */
         if ((!*auxOne) && (!*auxTwo))
         {
-            return 0;
+            return LEFT_EQ_RIGHT;
         }
 
         /* whichever version still has characters left over wins */
         if (!*auxOne)
         {
-            return -1;
+            return RIGHT_IS_NEWER;
         }
         else
         {
-            return 1;
+            return LEFT_IS_NEWER;
         }
     }
 
+    /**
+     *  Compare two EVR values to determine which is newer
+     *
+     *  This method compares the epoch, version, and release of the
+     *  provided package strings, assuming that epoch is 0 if not provided.
+     *  Comparison is performed on the epoch, then the version, and then
+     *  the release. If at any point a non-equality is found, the result is
+     *  returned without any remaining comparisons being performed (e.g. if
+     *  the epochs of the packages differ, the versions are releases are
+     *  not compared).
+     *
+     * @see rpmdev-vercmp(1)
+     *
+     * @param rRpmVersion The second RPM version.
+     *
+     * @retval 0 If a and b are equal.
+     * @retval <0 If a is smaller than b.
+     * @retval >0 If a is greater than b.
+     */
+    int compareRpmVersion(const uint32_t rEpoch, const std::string& rVersion, const std::string& rRelease) const
+    {
+        // Compare epochs
+        if (m_epoch != rEpoch)
+        {
+            return m_epoch < rEpoch ? RIGHT_IS_NEWER : LEFT_IS_NEWER;
+        }
+
+        // If epochs are equal, compare versions.
+        auto verComp = compareVersions(m_version, rVersion);
```

Comment:
Nitpick: Can use const
```suggestion
        const auto verComp = compareVersions(m_version, rVersion);
```

Reply:
Done!